### PR TITLE
feat: kerberos_renewal_principals service

### PR DESCRIPTION
**Pull request by Johaney-s**
* This service returns all kerberos principals from valid users

### CODE REVIEW

**Review by metodej — APPROVED** [review]
The code looks good!

File: gen/kerberos_renewal_principals [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+local $::SERVICE_NAME = basename($0);
+local $::PROTOCOL_VERSION = "3.0.0";
+my $SCRIPT_VERSION = "3.0.0";
+
+perunServicesInit::init;
+my $data = perunServicesInit::getHashedHierarchicalData;
+my $DIRECTORY = perunServicesInit::getDirectory;
+
+
+our $A_PRINCIPALS;                    *A_PRINCIPALS =                       \'urn:perun:user:attribute-def:virt:kerberosLogins';
```
Comment (zlamalp) [inline gen/kerberos_renewal_principals:18]: Service required attribute should be `user:def:kerberosLogins`

File: slave/process-kerberos-renewal-principals/changelog [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (zlamalp) [inline slave/process-kerberos-renewal-principals/changelog:1]: Changelog must be filled with first version for the package to be built, so something like this should work:
```
perun-slave-process-kerberos-renewal-principals (3.0.0) stable; urgency=low

  * added new service kerberos-renewal-principals

 -- Johana Supikova <xsupikov@fi.muni.cz>  Wed, 11 Aug 2021 10:37:00 +0200
```

**Review by zlamalp — CHANGES_REQUESTED** [review]

File: gen/kerberos_renewal_principals (hunk 1 above)
Comment (Johaney-s) [reply gen/kerberos_renewal_principals]: Switched `virt` to `def`.

**Review by zlamalp — CHANGES_REQUESTED** [review]
Can you please rebase it? It will require new version in changelog for meta package.

**Comment by Johaney-s** [comment]
> Can you please rebase it? It will require new version in changelog for meta package.

Rebased and updated changelog version.

**Review by zlamalp — APPROVED** [review]

**Pull request merged**